src/lightning/fabric/accelerators/tpu.py

```diff
@@ -49,14 +49,21 @@ def parse_devices(devices: Union[int, str, List[int]]) -> Union[int, List[int]]:
     @staticmethod
     def get_parallel_devices(devices: Union[int, List[int]]) -> List[torch.device]:
```

Comment:
It would be nice if we could adjust the existing tests or add new ones under tests/test_fabric/accelerators/test_tpu.py for the changes here to also cover pjrt.

Reply:
Once https://github.com/Lightning-AI/lightning/pull/17368 lands, we'll have PJRT testing in place